### DISCUSSION TITLE
Is it possible to disable wrapping for a piece of text?

### DISCUSSION THREAD

**Discussion by brandoncc** [prawnpdf/prawn#1330] [General]
I need to produce lines of text which can wrap between the lines, but not within them. The equivalent would be:

```html
<span style="word-wrap: nowrap;">This is the first line</span>
<span style="word-wrap: nowrap;">This is the second line</span>
```

I tried replacing all whitespace in the sentences with `Prawn::Text::NBSP` but the result was that it split in the middle of words in the sentence.

**Comment by pointlessone** [comment]
You can use `draw_text` to bypass text fragmentation. It only draws text.

```ruby
# This is going to be broken to fit bounds.
text "This is the first line. " * 10

move_down 10

# This is going to be a single line of text that goes outside of bounds and even the page.
draw_text "This is the second line. " * 10, at: [0, cursor]
```
